scripts/completion/dvc.zsh

```diff
@@ -72,8 +72,9 @@ _dvc_cache=(
 
 _dvc_checkout=(
   {-d,--with-deps}"[Checkout all dependencies of the specified target.]"
-  {-f,--force}"[Do not prompt when removing working directory files.]"
   {-R,--recursive}"[Checkout all subdirectories of the specified directory.]"
+  {-f,--force}"[Do not prompt when removing working directory files.]"
+  {--relink}"[Recreate links or copies from cache to workspace.]"
```

Comment:
This is giving me
```console
$ dvc checkout - [TAB]
_arguments:comparguments:325: invalid argument: {--relink}[Recreate links or copies from cache to workspace.]
_arguments:comparguments:325: invalid argument: {--relink}[Recreate links or copies from cache to workspace.]
_arguments:comparguments:325: invalid argument: {--relink}[Recreate links or copies from cache to workspace.]
```

@Suor

Reply:
I'm on ZShell BTW. Is it just me or is something missing from the script?

Reply:
Can you create a new issue for this @jorgeorpinel ?

Reply:
Oh, well I already reopened the original one (see https://github.com/iterative/dvc/issues/2983#issuecomment-570462666) and then it was fixed by Ramon. All good, thanks!
p.s. please try to notice my mentions in the dvc.org repo, maybe you have notifications off? Thanks @Suor